### DISCUSSION TITLE
Does one need to run a full node or pruned node to accept payments in a cryptocurrency?

### DISCUSSION THREAD

**Discussion by trymeouteh** [MrNaif2018/bitcart#277] [Q&A] [answered]
Does one need to run a full node or pruned node of the blockchain to accept payments in a cryptocurrency or can one not run a blockchain node at all and not require lots of storage space to use bitcartcc?

**Comment by MrNaif2018** [comment, marked as answer]
You can just use BitcartCC, we use lightweight solutions which are almost as secure as a full node. For btc we use electrum wallet, for eth we use geth full node in light syncmode. But if needed you can always point it out to your full node

**Reply by trymeouteh** [reply]
So no need to run any blockchain node at all for any coin?

**Reply by MrNaif2018** [reply]
By default no need to run any node, it will handle it for you. But there is COIN_SERVER setting where you can point it to your own electrumx server or geth full node
